test/cbmc/proofs/decodeBase64IndexBuffer/decodeBase64IndexBuffer_harness.c

```diff
@@ -39,11 +39,11 @@
 #define MAX_NUM_BASE64_DATA            4U
 
 /* Declaration of the mangled name static function which is created by CBMC for static functions. */
-Base64Status_t __CPROVER_file_local_ota_base64_c_decodeBase64IndexBuffer( uint8_t * pDest,
-                                                                          const size_t destLen,
-                                                                          size_t * pResultLen,
+Base64Status_t __CPROVER_file_local_ota_base64_c_decodeBase64IndexBuffer( uint32_t * pDest,
+                                                                          uint32_t * destLen,
                                                                           const uint8_t * pEncodedData,
-                                                                          const size_t encodedLen );
+                                                                          const size_t encodedLen,
+                                                                          size_t * pResultLen );
```

Comment:
Do the parameter names need to match the [static function](https://github.com/aws/ota-for-aws-iot-embedded-sdk/blob/2557f84b8295a30306aced925048d1073dcde968/source/ota_base64.c#L318-L322) which CBMC is overriding?

Reply:
The names do not need to match, just the types (note that previously several parameters were missing.